src/ui/main.cpp

```diff
@@ -23,7 +32,150 @@ bool          running     = true;
 
 static void   main_loop(void *);
 
-int           main(int, char **) {
+ImFont       *addDefaultFont(float pixel_size) {
+    ImGuiIO     &io = ImGui::GetIO();
+    ImFontConfig config;
+    config.SizePixels = pixel_size;
+    // high oversample to have better looking text when zooming in on the flowgraph
+    config.OversampleH = config.OversampleV = 4;
+    config.PixelSnapH                       = true;
+    ImFont *font                            = io.Fonts->AddFontDefault(&config);
+    return font;
+}
+
+class SumBlock : public DigitizerUi::Block {
```

Comment:
Move `SumBlock` and `FFTBlock` into separate files and maybe move them and datasource, datasink and flowgraph into a flowgraph subdir?

Reply:
Done